.github/workflows/run-archiver.yml

```diff
@@ -17,24 +14,10 @@ jobs:
     strategy:
       matrix:
         dataset:
-          - censusdp1tract
-          - eia176
           - eia860
```

Comment:
860 will almost always report no change except for when annual data is updated, but I think it's ok to run once a quarter anyways!